components/Faq/FAQ.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from "react";
+
+function FAQ({ faq: { question, answer }, index }) {
+  const [open, setOpen] = useState(false);
+  return (
+    <div className={`faq-item ${open ? 'open' : ''}`} key={index} onClick={() => setOpen(!open)}>
+      <div className="question-container"> 
+        <div className="faq-question">{question}</div> 
+        <span className="icon-arrow--down" />
+      </div>
+      <div className="faq-answer">{answer}</div>
+      <style jsx>
+        {`
+          { /* STYLES FOR MOBILE */ }
+          @media only screen and (max-width: 750px) {
+            .faq-item {
+              margin: 15px;
+              padding: 15px;
+              background-color: #FFF;
+              border-radius: 4px;
+              box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.2);
```

Comment:
Para empatar la sombra con los demás componentes
```suggestion
              box-shadow: 0px 10px 30px rgba(0,0,0,0.12);
```